component/main.jsonnet

```
@@ -100,6 +100,87 @@ local canary = kube._Object('monitoring.appuio.io/v1beta1', 'SchedulerCanary', '
   },
 };
 
+local storageCanaries = std.flattenArrays(std.filterMap(
+  function(storageclass) params.slos.storage.canary._sli.volume_plugins[storageclass] != null,
+  function(storageclass)
+    local p = params.slos.storage.canary._sli.volume_plugins_default_params + com.makeMergeable(params.slos.storage.canary._sli.volume_plugins[storageclass]);
+    local manifestName = 'canary-%s' % if storageclass == '' then 'default' else storageclass;
```

Maybe prefix with `storage-canary` instead of `canary`?

```suggestion
    local manifestName = 'storage-canary-%s' % if storageclass == '' then 'default' else storageclass;
```

---

Good idea especially since the default storage class canary is just called `canary-default`.